cmd/hamctl/command/rollback.go

```diff
@@ -32,31 +32,50 @@ has no effect.`,
 			})
 		},
 		RunE: func(c *cobra.Command, args []string) error {
-			committerName, committerEmail, err := git.CommitterDetails()
-			if err != nil {
-				return err
-			}
-			var resp httpinternal.RollbackResponse
-			path, err := client.URL("rollback")
-			if err != nil {
-				return err
+			var currentRelease httpinternal.DescribeReleaseResponseRelease
+			var rollbackTo *httpinternal.DescribeReleaseResponseRelease
+
+			if artifactID == "" {
+				releases, err := actions.ReleasesFromEnvironment(client, *service, environment, 2)
+				if err != nil {
+					return err
+				}
+
+				if len(releases) < 2 {
+					return fmt.Errorf("can't do rollback, because there isn't a release to rollback to")
+				}
+				currentRelease = releases[0]
+				rollbackTo = &releases[1]
+			} else {
+				releases, err := actions.ReleasesFromEnvironment(client, *service, environment, 10)
```

Comment:
Yeah we could, but wasn't sure if it made sense. You sure the magic 10 wont be good enough in most cases? Otherwise maybe my other change (not yet in a PR) on a terminal prompt will eliminate this need.